docs/references/express/available-methods.mdx

```diff
@@ -0,0 +1,89 @@
+---
+
+# Express.js Available Methods
+
+All resource operations are mounted as sub-APIs on the `clerkClient` object. You can find the full list of available operations in the [JavaScript Backend SDK](/docs/references/backend/overview) documentation. To access a resource, you must first instantiate a `clerkClient` instance.
+
+## Instantiate a default `clerkClient` instance
+
+If you would like to use the default instance of `clerkClient` provided by the SDK, you can provide the `CLERK_SECRET_KEY` as an environment variable and instantiate `clerkClient` without passing configuration options.
+
+<CodeBlockTabs options={["ESM", "CJS"]}>
+
+  ```js
+  import { clerkClient } from '@clerk/express';
+
+  const userList = await clerkClient.users.getUserList();
+  ```
+
+  ```js
+  const pkg = require('@clerk/express');
+  const clerkClient = pkg.default;
+
+  clerkClient.sessions
+    .getSessionList()
+    .then((sessions) => console.log(sessions))
+    .catch((error) => console.error(error));
+  ```
+
+</CodeBlockTabs>
+
+## Instantiate a custom `clerkClient` instance
+
+If you would like to customize the behavior of the SDK, you can instantiate a `clerkClient` instance yourself by calling `createClerkClient` and passing in [options](#customizing-resources).
+
+The example below shows how to use `createClerkClient` to create a `clerkClient` instance and pass a Clerk secret key *instead of* setting a `CLERK_SECRET_KEY` environment variable.
+
+<InjectKeys>
+
+<CodeBlockTabs options={["ESM", "CJS"]}>
+```js
+import { createClerkClient } from '@clerk/express';
+
+const clerkClient = createClerkClient({ secretKey: '{{secret}}' });
+
+const clientList = await clerkClient.clients.getClientList();
+```
+
+```js
+const { createClerkClient } = require('@clerk/express/cjs');
+
+const clerkClient = createClerkClient({ secretKey: '{{secret}}' });
+
+clerkClient.clients
+  .getClientList()
+  .then((clientList) => console.log(clientList))
+  .catch((error) => console.error(error));
+```
+</CodeBlockTabs>
+
+</InjectKeys>
+
+### Customizing resources
+
+The following options are available for you to customize the behaviour of the `clerkClient` object.
+
+<Callout type="info">
+  Most options can also be set as environment variables so that you don't need to pass anything to the constructor.
+</Callout>
+
+| Option | Description | Default | Environment variable |
+| --- | --- | --- | --- |
+| `secretKey` (required) | Server key for `api.clerk.com`. Can be found in your Clerk Dashboard on the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys)** page. | none | `CLERK_SECRET_KEY` |
+| `apiVersion` | API version to use | `v4` | `CLERK_API_VERSION` |
+| `apiUrl` | For debugging | `https://api.clerk.com` | `CLERK_API_URL` |
+
+{/* TODO: I think more of a description and example can be added for httpOptions. Also, we need to add information for audience, proxyUrl, jwtKey, publishableKey, domain, isSallite, telemetry */}
```

Comment:
🙃 We should remove this TODO and add a reference to the [backend SDK](https://clerk.com/docs/references/backend/overview) where the `#customizing-resources` is used in this docs or in Nextjs/Remix/... a customizing resources section exists.

The reference could be a different menu item describing when `createClerkClient()` should be used and the arguments (optional / required).

Since this could be considered a refactoring let's add the arguments here and mark them as optional / required and we can open a ticket in the docs team to proceed with the refactoring if it seems right.

cc: @alexisintech

Reply:
yes, this was in the works and I have published it https://github.com/clerk/clerk-docs/pull/1246
We can replace the clerk node sdk stuff in clerk/docs#1246 with the new clerk/express sdk stuff - and once its merged, update this PR accordingly to use the new info